### PR TITLE
Add reader function for Raw FIF files

### PR DESCRIPTION
**Pull request by teonlamont**

### CODE REVIEW

**Comment by teonlamont** [comment]
addresses #1880

File: mne/io/fiff/raw.py [hunk 1]
```diff
@@ -621,3 +622,61 @@ def _check_raw_compatibility(raw):
...
+        If set to n, e.g. 3, apply gradient compensation of grade n as
+        for CTF systems.
+    add_eeg_ref : bool
+        If True, add average EEG reference projector (if it's not already
+        present).
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Attributes
+    ----------
```
Comment (agramfort) [inline mne/io/fiff/raw.py]: Bad docstring. Should be returns of a RawFIFF instance

Also update a test. Not à new test

**Comment by agramfort** [comment]
LGTM

**Comment by dengemann** [comment]
Shall we call it *_fif or *_fiff ?

**Comment by agramfort** [comment]
I'd go for .fif as extension is .fif

**Comment by teonlamont** [comment]
+1

-teon

**Comment by agramfort** [comment]
+1 for merge.

Shall we rename RawFIFF to RawFIF for consistency?

**Comment by dengemann** [comment]
> Shall we rename RawFIFF to RawFIF for consistency?

and then also the io.fiff to io.fif ??

No release cycle without deprecation fun :)

**Comment by dengemann** [comment]
Thanks @teonlamont!

**Comment by agramfort** [comment]
and then also the io.fiff to io.fif ??

**Comment by agramfort** [comment]
and then also the io.fiff to io.fif ??

No.

**Pull request merged**